### PR TITLE
feat(torch-frontend): Added cdist

### PR DESCRIPTION
**Pull request by teckno**
# PR Description

## Related Issue

Close https://github.com/unifyai/ivy/issues/21894

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @teckno 

I ran your tests and I'm getting these failures
Would you look into this and run the tests locaaly to cross verify
image

File: ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py [hunk 1]
```diff
@@ -510,6 +510,46 @@ def test_torch_cartesian_prod(
     )
 
 
+# cdist
+@handle_frontend_test(
+    fn_tree="torch.cdist",
+    dtypes_and_x=helpers.dtype_and_values(
+        available_dtypes=[
```
Comment (YushaArif99) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py:517]: why only limit to `float32` and `float64`? I think you should use `avalaible_dtypes=valid` here

File: ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py [hunk 2]
```diff
@@ -510,6 +510,46 @@ def test_torch_cartesian_prod(
...
+    frontend,
+    fn_tree,
+    on_device,
+    test_flags,
+    backend_fw,
+):
+    input_dtypes, x = dtypes_and_x
+    p = 2.0
+    helpers.test_frontend_function(
+        input_dtypes=["float64"],
```
Comment (YushaArif99) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py:541]: shouldn't this be `input_dtypes=input_dtypes` ?

**Comment by Sarvesh-Kesharwani** [comment]
Hi @teckno 
please resolve the merge conflicts.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by Ishticode** [comment]
Thank you very much for the PR. Closing this due to lack of activity with changes requested remaining unresolved for over 6 months. Please feel free to reopen if you would like to continue working on this.  :)

**Pull request closed without merging**